experimental/packages/opentelemetry-exporter-prometheus/test/PrometheusExporter.test.ts

```diff
@@ -261,8 +261,9 @@ describe('PrometheusExporter', () => {
 
     beforeEach(done => {
       exporter = new PrometheusExporter({}, () => {
-        meterProvider = new MeterProvider();
-        meterProvider.addMetricReader(exporter);
+        meterProvider = new MeterProvider({
```

Comment:
This looks weird to me that we have to create a MeterProvider in the callback of a PrometheusExporter.

I submitted a follow-up at https://github.com/open-telemetry/opentelemetry-js/pull/4431.